introduction.md

```diff
@@ -0,0 +1,8 @@
+Hi, I'm randy, working my way through codeschool.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.